sanitizers/src/main/java/jaz/Zer.java

```diff
@@ -29,6 +29,9 @@
  */
 @SuppressWarnings("unused")
 public class Zer implements java.io.Serializable {
+
+  public static final String HONEYPOT_CLASS_NAME = "jaz.Zer";
+
```

Comment:
This is dangerous: We should not reference `jaz.Zer` from anywhere explicitly as it might crash or confuse the sanitizers. Could we move this constant into a Utils file in the sanitizers package?